pkg/resources/nlb_service/datasource_list.go

```diff
@@ -129,7 +129,7 @@ Corresponding resource: [exoscale_nlb](../resources/nlb.md).`,
 				Optional:            true,
 			},
 			NLBServiceListAttrNLBServiceList: schema.ListNestedAttribute{
-				MarkdownDescription: "The list of [exoscale_nlb_service](./nlb_service.md).",
+				MarkdownDescription: "The list of [exoscale_nlb_service](./nlb_service_list.md).",
```

Comment:
Not the best wording in description, but it should be `nlb_service.md` because it's a list of `nlb_service` elements.

Reply:
I'm just doing a mechanical change to make all the links in the doc resolve with the current file names.

If we're not happy with the current file structure or descriptions, we can tackle that in a follow up PR.